body.tex

```diff
@@ -352,9 +413,10 @@ \subsubsection{Bucket Epics} \label{sec:bucket}
 As such, we acknowledge that they are necessary, but seek to minimize the fraction of our resources assigned to them.
 If more than a relatively small fraction of the work for a \gls{cycle} is assigned to bucket \glspl{epic}, please consider whether this is really necessary and appropriate.
 
-Be aware that even bucket \glspl{epic} must be assigned to a specific \emph{leaf} \gls{element} of the \gls{WBS}.
-That is, it is not in general possible to define an \gls{epic} which handles bug reports or emergent feature requests across the whole of the codebase unless a specific \gls{WBS} leaf \gls{element} is devoted to maintenance activities of this type.
-Instead, it may be necessary to define a different bucket \gls{epic} for each leaf of the \gls{WBS} tree.
+% I think this is a tie to the Construction EVMS and is not needed here.
```

Comment:
At some point you may like this to know which WBS we spend more on .. but indeed its like this construction for EVMS.